dockerfiles/alpine/analytics-dashboard/Dockerfile

```diff
@@ -36,12 +36,7 @@ ARG WSO2_SERVER=${WSO2_SERVER_NAME}-${WSO2_SERVER_VERSION}
 ARG WSO2_SERVER_HOME=${USER_HOME}/${WSO2_SERVER}
 ARG WSO2_SERVER_DIST_URL=https://github.com/wso2/${WSO2_SERVER_REPOSITORY}/releases/download/v${WSO2_SERVER_VERSION}/${WSO2_SERVER}.zip
 # build argument for MOTD
-ARG MOTD='printf "\n\
- Welcome to WSO2 Docker Resources \n\
- --------------------------------- \n\
- This Docker container comprises of a WSO2 product, running with its latest GA release \n\
- which is under the Apache License, Version 2.0. \n\
- Read more about Apache License, Version 2.0 here @ http://www.apache.org/licenses/LICENSE-2.0.\n"'
+ARG MOTD
```

Comment:
@NishikaDeSilva IMO, we don't need to change this since the default open source code should point to the details of Apache Licensing. When building images containing WSO2 Updates, we override this value with the WSL based MOTD.

Please correct me if I am wrong, @NishikaDeSilva @DilanUA @ThilinaManamgoda.